.bazelrc

```diff
@@ -98,3 +98,8 @@ build:clang-tidy --copt=-Wunused-private-field --copt=-Wuser-defined-warnings
 # Support user-provided user.bazelrc
 try-import %workspace%/user.bazelrc
     .
+
+# Enable rustfmt tests on all Rust targets
+build --@rules_rust//:rustfmt.toml=//tools/rust:rustfmt.toml
+build --aspects=@rules_rust//rust:defs.bzl%rustfmt_aspect
+build --output_groups=+rustfmt_checks
```

Comment:
This should probably be limited to the prepush config, no?

Reply:
And certainly come before the try-import for user.bazelrc (which should always come last)

Reply:
IMO this is better because it will come up when running `bazel build` during local development

Reply:
I'm dubious that it should be part of the default configuration (rather than its own), but am not particularly bothered by it.

But it should definitely come before the `try-import user.bazelrc`.